gunicorn_conf.py

```diff
@@ -0,0 +1,5 @@
+"""Gunicorn production config."""
+import multiprocessing
```

Comment:
I think we should try to keep all our Python files 2/3-agnostic (or none of them), so proposing to add the usual one; also fixed in #184